src/test/java/api/support/builders/CheckInByBarcodeRequestBuilder.java

```diff
@@ -34,6 +39,7 @@ public JsonObject create() {
     put(request, "itemBarcode", this.itemBarcode);
     put(request, "checkInDate", this.checkInDate);
     put(request, "servicePointId", this.servicePointId);
+    put(request, CLAIMED_RETURNED_RESOLVED_BY, this.claimedReturnedResolvedBy);
```

Comment:
Is this constant from the production code?

Reply:
Yes, it is from the CheckInByBarcodeRequest class.

Reply:
I would prefer that the API test code does not depend upon production code. I think this kind of reuse weakens the tests, partly for readability / documentation and partly because the tests won't  identify when the API has changed or the constant has become incorrect.